main.py

```diff
@@ -29,5 +29,9 @@
         from scoreboard.tests import data
         models.db.create_all()
         data.create_all()
+    elif 'runtests' in sys.argv:
+        from scoreboard.tests.unit import test_all
```

Comment:
This isn't going to work with the new module name.  We should have a central test runner, but I don't know the best way to do that.